rust/connlib/libs/tunnel/src/tun_darwin.rs

```diff
@@ -1,22 +1,22 @@
 use ip_network::IpNetwork;
 use libc::{
-    close, connect, ctl_info, fcntl, getsockopt, ioctl, iovec, msghdr, recvmsg, sendmsg, sockaddr,
-    sockaddr_ctl, sockaddr_in, socket, socklen_t, AF_INET, AF_INET6, AF_SYSTEM, AF_SYS_CONTROL,
-    CTLIOCGINFO, F_GETFL, F_SETFL, IF_NAMESIZE, IPPROTO_IP, O_NONBLOCK, PF_SYSTEM, SOCK_DGRAM,
+    close, ctl_info, fcntl, getpeername, getsockopt, ioctl, iovec, msghdr, recvmsg, sendmsg, sockaddr,
+    sockaddr_ctl, sockaddr_in, socket, socklen_t, AF_INET, AF_INET6, AF_SYSTEM,
+    CTLIOCGINFO, F_GETFL, F_SETFL, IF_NAMESIZE, IPPROTO_IP, O_NONBLOCK,
     SOCK_STREAM, SYSPROTO_CONTROL, UTUN_OPT_IFNAME,
 };
 use libs_common::{CallbackErrorFacade, Callbacks, Error, Result, DNS_SENTINEL};
 use std::{
     ffi::{c_int, c_short, c_uchar},
     io,
-    mem::{size_of, size_of_val},
+    mem::size_of,
     os::fd::{AsRawFd, RawFd},
     sync::Arc,
 };
 
 use super::InterfaceConfig;
 
-const CTRL_NAME: &[u8] = b"com.apple.net.utun_control";
+const CTL_NAME: &[u8] = b"com.apple.net.utun_control";
```

Comment:
I renamed this constant to be slightly more aligned with [Apple's documentation](https://developer.apple.com/documentation/kernel/kern_ctl_reg/1809212-ctl_name).